src/index.js

```diff
@@ -0,0 +1,80 @@
+import './style.css';
+import { countShows } from './modules/counterItem.js';
+import createShowCard from './modules/display.js';
+
+const API = 'https://api.tvmaze.com/shows/1/episodes';
+const likesAPI = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/OpXkQhwbfD4wnLSWy6wV/likes';
+
+const displayEpisode = document.querySelector('.episode');
+const previous = document.querySelector('.previous');
+const next = document.querySelector('.next');
+const pageNum = document.querySelector('.page-numbers');
+const showsHeader = document.querySelector('.shows-header');
+const body = document.querySelector('body');
+
+let showsArray = [];
+
+const getData = async (API) => {
+  const result = await fetch(API);
+  return result;
+};
+
+const displayShows = (shows, pageNumber, heading) => {
+  shows.slice(pageNumber * 10 - 10, pageNumber * 10).forEach((show) => {
+    const div = createShowCard(show);
+    displayEpisode.append(div);
+  });
+
+  const showsCount = countShows(showsArray);
+  heading.innerHTML = `Episodes: ${showsCount}`;
+};
+
+function loadNext(pageNumber, shows, showsHeader) {
+  const nextPage = pageNumber + 1;
+  if (nextPage < 5) {
+    pageNum.innerHTML = nextPage;
+    displayEpisode.innerHTML = '';
+    getData(API)
+      .then((response) => response.json())
+      .then((json) => displayShows(json, pageNumber + 1));
+    displayShows(shows, nextPage, showsHeader);
+  }
+}
+
+function loadPrevious(pageNumber, shows, showsHeader) {
+  const previousPage = pageNumber - 1;
```

Comment:
- [ ] Same here,  Please, It's one of the project requirements you use `arrow` syntax for functions instead of JavaScript Function Expressions because you learned how to use ES6 new features. Kindly make this :wink: 💯 

- For more documentation, you can visit this [link](https://www.w3docs.com/learn-javascript/function-expressions.html)

Reply:
Done